lib/compilers/postcss-compiler.js

```diff
@@ -0,0 +1,28 @@
+const postcss = require('postcss')
+const postcssrc = require('postcss-load-config')
+const ctx = { parser: true, map: 'inline' }
+const { plugins } = postcssrc.sync(ctx)
+
+const throwError = require('../throw-error')
+
+module.exports = (content) => {
+  let css = null
+
+  postcss(plugins)
```

Comment:
You can run postcss synchronously: https://github.com/vuejs/component-compiler-utils/blob/37b4a6a99ecaa8bf27c777191ac17e9168c70cd1/lib/compileStyle.ts#L115

It means asynchronous plugins won't be applied, but I think that's is fine

Reply:
I added check for async plugin, i think it is more correct way, because we have async plugin and processing postcss is crashed